opencve/templates/cve.html

```diff
@@ -220,6 +220,9 @@ <h3 class="box-title">Tags</h3>
                     <p><strong>Published :</strong> {{ cve.created_at.strftime('%Y-%m-%d %H:%M') }}</p>
                     <p><strong>Updated :</strong> {{ cve.updated_at.strftime('%Y-%m-%d %H:%M') }}</p>
                     <hr/>
+                    <p><strong><i class="fa fa-external-link"></i> NVD link :</strong> <a
```

Comment:
Thanks for this PR, unfortunately your copy-paste was too fast ^^ Please change `NVD link` with `CVE.org link`.

You can also switch it to the end of the list (to keep the MITRE links grouped):

<img width="272" alt="image" src="https://user-images.githubusercontent.com/1552526/221690772-fd841479-5573-4165-973e-27cf0742565e.png">